frontend/src/metabase/visualizations/visualizations/Text/Text.jsx

```diff
@@ -46,6 +47,9 @@ export function Text({
   const justAdded = useMemo(() => dashcard?.justAdded || false, [dashcard]);
   const [textValue, setTextValue] = useState(settings.text);
 
+  const tc = useTranslateContent();
+  const translatedText = useMemo(() => tc(settings.text), [settings.text, tc]);
```

Comment:
Should you be using `textValue` here from state instead of what's in the prop?

Reply:
That might make sense but just to leave the logic as unchanged as possible, I used settings.text here, since it's what we used below on line 68. I'm also not localizing the editable state of this component, only the readonly state, so I think that's why I don't need to touch `textValue`